### DISCUSSION TITLE
Big Sur Icon

### DISCUSSION THREAD

**Discussion by alexanderjamesrohrig** [quicktype/quicktype#1604] [Ideas]
The QuickType app icon for macOS no longer conforms to Apple's Human Interface Guidelines.

Starting with Big Sur, apps are supposed to use a rounded-rectangle shape.

[Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/)

**Comment by dvdsgl** [comment]
Great idea! We would love for someone to contribute it.
